project/dependencies.scala

```diff
@@ -1,7 +1,7 @@
 import sbt._
 
 object Dependencies {
-  val capiVersion = "12.10"
+  val capiVersion = "12.18"
```

Comment:
So if I'm right - this is the key change here. And this new capiVersion of the libraries below now has the _old_ libthrift version that doesn't have a horrible backwards noncompatible breaking change?

Reply:
Yes, precisely.

Reply:
cool